feat: show wakatime stats in percentage (resolves #3016) (#3326)
anuraghazra/github-readme-stats: readme.md

*   `layout` - Switches between two available layouts `default` & `compact`.  Default `default`.
*   `langs_count` - Limits the number of languages on the card, defaults to all reported languages *(number)*.
*   `api_domain` - Sets a custom API domain for the card, e.g. to use services like [Hakatime](https://github.com/mujx/hakatime) or [Wakapi](https://github.com/muety/wakapi) *(string)*. Default `Waka API`.

> * `display_format` - Option to display wakatime stats in percentage or time. Default `time`.

Reviewer: ```suggestion
* `display_format` - Changes the wakatime stats display format (i.e. `time` or `percent`). Default `time`.
```

> * `display_format` - Option to display wakatime stats in percentage or time. Default `time`.

Reviewer: Let's mention the options.

```suggestion
* `display_format` - Modifies the presentation style of WakaTime statistics. Choose between `time` to display time-based stats or `percent` to show percentages. The default is set to time.
```

> * `display_format` - Sets the WakaTime stats display format. Choose `time` to display time-based stats or `percent` to show percentages. The default is set to `time`.

Reviewer 2: Let's replace ``The default is set to `time`.`` with ``Default: `time`.``
